firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -85,8 +85,9 @@ public static AppStateMonitor getInstance() {
 
   private ApplicationProcessState mCurrentState = ApplicationProcessState.BACKGROUND;
 
-  private Set<WeakReference<AppStateCallback>> mClients =
+  private Set<WeakReference<AppStateCallback>> appStateClients =
```

Comment:
Can we use the term subscribers? may be activityStateSubscribers, appStateSubscribers? Client is too generic a term.

Reply:
Done.